### PR TITLE
add target to the cookie-link

### PR DESCRIPTION
**Pull request by F4b1n0u**
to allow the target _blank (new tab)

### CODE REVIEW

File: src/CookieBanner.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ const propTypes = {
   onAccept: React.PropTypes.func,
   link: React.PropTypes.shape({
     msg: React.PropTypes.string,
-    url: React.PropTypes.string.isRequired
+    url: React.PropTypes.string.isRequired,
+    target: _react2['default'].PropTypes.oneOf(['_blank', '_self', '_parent', '_top', 'framename'])
```
Comment (gabro) [inline src/CookieBanner.js]: this doesn't look right

File: src/CookieBanner.js [hunk 2]
```diff
@@ -115,6 +116,7 @@ export default React.createClass({
       return (
         <a
           href={link.url}
+          target: link.target,
```
Comment (gabro) [inline src/CookieBanner.js]: invalid syntax

**Comment by gabro** [comment]
thanks @F4b1n0u, that's a useful addition 👍 

However the diff doesn't look right, you probably committed the transpiled version.

Can you fix it so that we can merge this?

**Comment by F4b1n0u** [comment]
Hi Mate
it's me who thanks you ;)
I branch master, happy my change commit and push ...
have I done something wrong ?

**Comment by F4b1n0u** [comment]
ha OK ... i'm on it :)

**Comment by F4b1n0u** [comment]
done :)

**Comment by gabro** [comment]
Thanks, but I think you still missed a spot. See my second comment on `invalid: true`

**Comment by F4b1n0u** [comment]
oops ... sorry about that :)

**Comment by F4b1n0u** [comment]
should be ok this time

**Comment by gabro** [comment]
great! Thanks 🎉

**Comment by gabro** [comment]
published as 0.0.12

**Pull request merged**